cmds/receive.c

```diff
@@ -1329,6 +1329,31 @@ static int process_fallocate(const char *path, int mode, u64 offset, u64 len,
 	return ret;
 }
 
+static int process_setflags(const char *path, int flags, void *user)
+{
+	int ret = 0;
+	struct btrfs_receive *rctx = user;
+	char full_path[PATH_MAX];
+	long ioc_flags = flags;
```

Comment:
Oops I was wrong, this is supposed to be an int after all: https://man7.org/linux/man-pages/man2/ioctl_iflags.2.html. So you can just pass `&flags` directly.